Library/Formula/python.rb

```diff
@@ -28,7 +29,7 @@ class Python < Formula
   depends_on "gdbm" => :recommended
   depends_on "openssl"
   depends_on "homebrew/dupes/tcl-tk" => :optional
-  depends_on "berkeley-db4" => :optional
+  depends_on "berkeley-db" => :optional
```

Comment:
That's a third-party extension module. This option builds a module named `bsddb`. https://docs.python.org/2/library/bsddb.html